### PR TITLE
Enhancement: Allow multiple task links in the PR description

### LINKED ISSUE #2: Allow more than one link
## Issue: 
Adding to the description more than one link to Teamwork task won't work.

## Expected behaviour
The action should identify more than one link and send one comment for each task link.

### PR DESCRIPTION
**Pull request by rafaeljusto**
Resolves #2

### CODE REVIEW

File: src/main.sh [hunk 1]
```diff
@@ -20,36 +20,39 @@ main() {
...
+    elif [ "$event" == "pull_request_review" ] && [ "$action" == "submitted" ]; then
+      teamwork::pull_request_review_submitted
+    elif [ "$event" == "pull_request_review" ] && [ "$action" == "dismissed" ]; then
+      teamwork::pull_request_review_dismissed
+    elif [ "$event" == "pull_request_review_comment" ] && [ "$action" == "deleted" ]; then
+      teamwork::pull_request_review_comment_deleted
+    else
+      log::message "Operation not allowed"
+      exit 0
+    fi
```
Comment (rafaeljusto) [inline src/main.sh:54]: We could move this to a function that would receive the `$event` and `$task_id`.

File: src/teamwork.sh [hunk 2]
```diff
@@ -2,12 +2,18 @@
...
-  task=$(echo "$body" | grep -Eo "$pat")
-  task_id=$(echo "$task" | tr -cd  '[:digit:]')
-
-  echo "$task_id"
+  local task_ids=()
+
+  pat='tasks\/([0-9]{1,})'
+  while [[ $body =~ $pat ]]; do
+    task_ids+=( "${BASH_REMATCH[1]}" )
+    body=${body#*"${BASH_REMATCH[1]}"}
```
Comment (miguelbemartin) [inline src/teamwork.sh:10]: This var (body) is a read-only var. You need to remove the `-r` from line 4.

File: src/main.sh (hunk 1 above)
Comment (miguelbemartin) [reply src/main.sh:54]: Maybe we can just move outside of the loop the lines 36-39.

I did today another PR removing some unused code, so the code inside the "for" won't be so long.

**Review by miguelbemartin — CHANGES_REQUESTED** [review]
It looks good. Only a small minor change.

**Comment by rafaeljusto** [comment]
@miguelbemartin Thanks for reviewing and detecting the issues! 😄 

I was checking the new [CONTRIBUTING](https://github.com/Teamwork/github-sync/blob/a2227733f679d222b5634c227e61be38e1b0d67f/CONTRIBUTING.md) file, and it's not clear to me how to test the code from this PR using the `act` tool. If I understood correctly I need to run it inside a repository that has the `.github/workflows/` files, but how would it link with the changes of this PR? 🤔  Do we have a special demo repository somewhere with the `events/pull_request_opened.json` file?

Thanks and sorry for about the many questions. First time playing with this 🙂 .

**Comment by miguelbemartin** [comment]
> @miguelbemartin Thanks for reviewing and detecting the issues! 😄
> 
> I was checking the new [CONTRIBUTING](https://github.com/Teamwork/github-sync/blob/a2227733f679d222b5634c227e61be38e1b0d67f/CONTRIBUTING.md) file, and it's not clear to me how to test the code from this PR using the `act` tool. If I understood correctly I need to run it inside a repository that has the `.github/workflows/` files, but how would it link with the changes of this PR? 🤔 Do we have a special demo repository somewhere with the `events/pull_request_opened.json` file?
> 
> Thanks and sorry for about the many questions. First time playing with this 🙂 .

I didn't get the time to finish the contributing guide. There are missing the files to simulate the Github events.

**Comment by rafaeljusto** [comment]
The changes were tested by Miguel locally and worked. Moving to a regular PR.

**Review by miguelbemartin — APPROVED** [review]
Tested locally again and it works as expected 👍 
Thanks, @rafaeljusto for your first contribution.

**Pull request merged**